ecommerce.yml

```diff
@@ -269,7 +269,7 @@ spec:
     spec:
       containers:
       - name: ecommerce-order
-        image: detcaccounts/ecommerce-order
+        image: gabelacework/ecommerce-order
```

Comment:
This change needs to be reverted before we merge the PR, just here for easier testing.